vizro-core/tests/unit/vizro/models/test_dashboard.py

```diff
@@ -75,7 +75,7 @@ def test_page_registry(self, vizro_app, page_1, page_2, mocker):
         mock_register_page.assert_any_call(
```

Comment:
I think we need a couple more test cases here (which might not pass yet). Let me add them and let's see what happens.

Reply:
Done  in [b427833](https://github.com/mckinsey/vizro/pull/1144/commits/b427833f2b56f08538b3f09c2f3edf8ccc12fb8f).